doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/data-sets-caching-con.adoc

```diff
@@ -20,7 +20,4 @@ When the cache is turned on, the data set is cached in a web browser during the
 
 When the cache is enabled, the {DECISION_ENGINE} caches the data set. This reduces the number of back-end requests to the remote storage system. All data set operations are performed in the {DECISION_ENGINE} using in-memory data. Enable back-end caching only if the data set size is not updated frequently and it can be stored and processed in memory. Using back-end caching is also useful in cases with low latency connectivity issues with the remote storage.
```

Comment:
I would mention somewhere where the caching can be set (Advanced tab)

Reply:
I have added the location of this feature in the content.